### ISSUE TITLE
VWAP calculation doesn't seem right

### ISSUE THREAD

**Issue by samjtro** [samjtro/go-algotrader#1]
I am used to the VWAP returning a value +-20% of the stock's Closing price, and yet I'm getting a decimal val 0-0.99. Not sure if this is an indication I am supposed to be doing something with this information. I need to do a bit more research into the calculation I've done, which is fairly straightforward (listed below). 

vwap = (Hi + Lo + Close) / Vol

**Comment by tanryberdi** [comment]
I will try to investigate for this issue, but i think i must start with you https://github.com/samjtro/go-tda, am i right? Because i want to run this repo, but i got `2022/08/09 14:39:13 Config File "tda-config" Not Found in ` message.

**Comment by samjtro** [comment]
Hmm. That's interesting; it should be working just fine. What platform are you working on?

As well, did you go through the instructions located in the README.md relating to the tda-config.env file? Essentially it's just a .env file for your TD APIKEY as well as your Time Difference for proper Timestamping. Required as the library uses the TDAmeritrade API, which is protected from Unauthorized Requests by said APIKEY.

**Comment by tanryberdi** [comment]
I am working on MacOS, i will go through README.md, btw, i have PR that updating `.gitignore` for Goland IDE

**Comment by samjtro** [comment]
Just merged! Appreciate the work on that. Yeah you should quickly run through it; it takes about 5-minutes to get set up and then you get infinite TD-Ameritrade requests through their API.

Basically, just create a file called "tda-config.env"; move it to your home directory; store the following information in it:
APIKEY=YourAPIKEY (found by creating an app at https://developer.tdameritrade.com/)
UTC_DIFF=YourUTCTimeDifference (e.g. for me that is -06:00 in Mountain Standard Time in the US)

**Comment by samjtro** [comment]
Fixed via commit: 9e62836baf7b221fe0e62fb149e02ca9a3297317

Thought I was doing something wrong and I was, the calculation for VWAP isn't `(Hi + Lo + Close) / 3` it's `(((Hi + Lo + Close) / 3) * Volume) / Volume`.

@tanryberdi - I ended up just wrapping this one up. I will be opening a few more issues for MACD & Chaikin Indicators. If you want the assignments just let me know, I'd be happy to give them to you.
